dev/src/types.ts

```diff
@@ -144,6 +144,16 @@ export interface FirestoreDataConverter<T> {
    */
   toFirestore(modelObject: T): DocumentData;
 
+  /**
+   * Called by the Firestore SDK to convert a custom model object of Partial<T>
+   * into a plain Javascript object (suitable for writing directly to the
+   * Firestore database).
+   *
+   * This method must be defined in order to use set() with objects of type
+   * Partial<T>.
```

Comment:
Please rewrite this part to refer to "merge". First paragraph is fine and dandy.

Reply:
done.